src/test/regress/input/multi_shard_modify.source

```diff
@@ -0,0 +1,135 @@
+--
+-- MULTI_SHARD_MODIFY
+--
+
+ALTER SEQUENCE pg_catalog.pg_dist_shardid_seq RESTART 102046;
+
+-- Create a new range partitioned customer_delete_protocol table and stage data into it.
+DROP TABLE IF EXISTS customer_delete_protocol;
+CREATE TABLE customer_delete_protocol (
+        c_custkey integer not null,
+        c_name varchar(25) not null,
+        c_address varchar(40) not null,
+        c_nationkey integer not null,
+        c_phone char(15) not null,
+        c_acctbal decimal(15,2) not null,
+        c_mktsegment char(10) not null,
+        c_comment varchar(117) not null);
+SELECT master_create_distributed_table('customer_delete_protocol', 'c_custkey', 'append');
+
+\STAGE customer_delete_protocol FROM '@abs_srcdir@/data/customer.1.data' with delimiter '|'
```

Comment:
Changed the test table, made it a simpler hash-distributed table. Instead of COPY, used INSERTs for test data and keep it in regular .sql/.out files. I liked it more now.